src/plugins/three/LoadRegionPlugin.d.ts

```diff
@@ -1,18 +1,18 @@
-import { TileBoundingVolume } from '../../../three/math/TileBoundingVolume';
-import { Tile } from '../../../base/Tile';
-import { TilesRenderer } from '../../../three/TilesRenderer';
+import { TileBoundingVolume } from '../../three/math/TileBoundingVolume';
+import { Tile } from '../../base/Tile';
+import { TilesRenderer } from '../../three/TilesRenderer';
 import { OBB } from '../../three/math/OBB';
 import { Sphere, Ray } from 'three';
 
-interface Region {
+declare class Region {
```

Comment:
Can you explain what "declare" is doing here?

Reply:
It only declares the class for the TypeScript compiler, not for the JavaScript runtime. It is required by TS to either have `declare` or `export` in top-level declarations. Based on your original code, I imagined you didn't want to export the class.